src/types/view/index.ts

```diff
@@ -68,6 +74,12 @@ export interface ViewClosedAction {
   api_app_id: string;
   token: string;
   is_cleared: boolean;
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
Haven't confirmed if the payload is actually getting these two keys yet in enterprise install (and that team is null)